shelley/chain-and-ledger/executable-spec/test/Examples.hs

```diff
@@ -131,19 +149,27 @@ alicePoolParams =
 
 -- | Helper Functions
 
+mkTestNoncePrf :: Seed -> Slot -> Seed
+mkTestNoncePrf eta0 s = (eta0 `seedOp` ss) `seedOp` SeedEta
+ where ss = slotToSeed s
 
-mkBlock :: Maybe HashHeader -> KeyPair -> (SKeyES, VKeyES) -> [Tx] -> Slot -> Seed -> Seed
-  -> UnitInterval -> Natural -> Block
-mkBlock prev cold (shot, vhot) txns s enonce bnonce l kesPeriod =
+mkTestLeaderPrf :: Seed -> Slot -> Seed
```

Comment:
maybe just a single function with the `SeedEta` / `SeedL` as additional parameter?

Reply:
they were only used once each anyway, so I just removed them and used let bindings instead